exercises/G-numbers/exercise.js

```diff
@@ -1 +1,9 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var NumberOfMentors= 8;
```

Comment:
me being picky, but JS variables should be camel case
i.e. they should start with a lower case letter